src/main/java/reposense/model/FileTypeManager.java

```diff
@@ -15,6 +18,8 @@
     private List<FileType> formats;
     private List<FileType> groups;
 
+    private final Logger logger = LogsManager.getLogger(this.getClass());
```

Comment:
Interesting approach here, but lets follow the convention we are using, for slightly better performance:
`private static final Logger logger = LogsManager.getLogger(FileTypeManager.class);`

Using your approach, we can remove the need to declare `private final Logger logger...` in every class, if we make it an interface? 🤔

Reply:
Will change back to use the existing convention.

>Using your approach, we can remove the need to declare private final Logger logger... in every class, if we make it an interface? 

I'm not really sure if that's possible, but for now let's stick to the existing convention